R/dev.R

```diff
@@ -0,0 +1,63 @@
+# Utility scripts for development purposes, that are not exported to users
+
+#' Update the metadata database in nectar using a newly created data frame
+#' @param metadata The data frame to upload
+#' @param version The version for the new metadata as a character scalar, e.g.
+#'   "0.2.3"
+#' @param credential_id The OpenStack application credential ID as a character
+#'   scalar. This is optional because you can alternatively source a
+#'   `-openrc.sh` file instead of providing it here.
+#' @param credential_id The OpenStack application credential secret as a
+#'   character scalar
+#' @noRd
```

Comment:
Can we please add an (even not evaluated) example here, so I can have the feeling of what is required and a description of what the input and output/behaviour is?